internal/jager-mongodb/reader.go

```diff
@@ -225,6 +225,19 @@ func (s *SpanReader) findTraceIDs(ctx context.Context, query *spanstore.TraceQue
 		"$lt": query.StartTimeMax,
 	}
 
+	// Filtering by concatenation of tags.
```

Comment:
isn't this already merged into the main branch? doing a rebase locally should pull those changes into your branch.

e.g. `git pull origin master --rebase`